v2/spanner-common/src/test/java/com/google/cloud/teleport/v2/spanner/migrations/avro/AvroTestingHelper.java

```diff
@@ -91,4 +118,17 @@ public static GenericRecord createIntervalRecord(Integer months, Integer hours,
     genericRecord.put("micros", micros);
     return genericRecord;
   }
+
+  public static GenericRecord createIntervalNanosRecord(
+      Long years, Long months, Long days, Long hours, Long minutes, Long seconds, Long nanos) {
+    GenericRecord genericRecord = new GenericData.Record(INTERVAL_NANOS_SCHEMA);
+    genericRecord.put("years", years);
```

Comment:
it would be good to move all these strings into constants. They seem to be all over the code

Reply:
The strings are specific to the schema. 
We will take up moving all the avro schema and logical type definitions to common code as a part of 
b/383689307

Currently the datastream package also uses string literals directly and that seems to have flown into the live changes.